rpc_jobs/standard_job_premerge.yml

```diff
@@ -0,0 +1,169 @@
+- project:
+    # This is the unique name to designate to the
+    # project. The name should relate to the git
+    # repository name for easier identification.
+    name: "sample"
+
+    # This is the URL for the project's repository
+    # which contains the scripts which will be
+    # executed.
+    repo_url: "https://github.com/rcbops/sample"
+
+    # This is the whitelist for branch names which
+    # should trigger the job automatically. Each
+    # item is used as a Java regex match.
+    branches:
+      - "master"
+
+    # This is a list of names to designate the image
+    # used when preparing a cloud instance. Each name
+    # must also include the key IMAGE with a value
+    # that corresponds to the appropriate image UUID
+    # or name of the image available in the cloud.
+    image:
+      - xenial:
+          IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
+
+    # This is a list of names to designate the 'scenario'
+    # which is being executed. This provides the ability
+    # to execute the same scripts with different values
+    # which affect the code path tested.
+    scenario:
+      - "default"
+
+    # This is a list of names to designate the 'action'
+    # which is executed. This provides the ability to
+    # execute different scripts.
```

Comment:
That could be misread, as the same entrypoint will be used for each scenario. This axis, allows the hook implementation to modify its behaviour based on the required action.

Reply:
Done.